cmd/params.go

```diff
@@ -87,12 +109,14 @@ func params(cmd *cobra.Command, args []string) {
 			log.Fatal("no subdirectories with corresponding ctl files found")
 		}
 	}
+
```

Comment:
Also, add .mod to the error message on line 109

Reply:
updated line 92: 

```
		ctlModels := utils.ListFilesByExt(utils.ListFiles(fi), "ctl")
		modModels := utils.ListFilesByExt(utils.ListFiles(fi), "mod")
		md := append(ctlModels, modModels...)
```

Created directory with :

```
 ls ~/Desktop/bbr/inst/model/nonmem/basic/test_batch_params
1       1.ctl   2       2.ctl   3       3.ctl   4       4.mod
```
Where 4.mod & dir 4 is the same as 3.ctl & dir 3 just renamed.
ran `bbi nonmem params --dir ~/Desktop/bbr/inst/model/nonmem/basic/test_batch_params` and getting expected output: 
```
dir,error,termination,THETA1,THETA2,THETA3,THETA4,THETA5,SIGMA(1_1),OMEGA(1_1),OMEGA(2_1),OMEGA(2_2),THETA9
/Users/AnnaN/Desktop/bbr/inst/model/nonmem/basic/test_batch_params/1,,0.00000E+00,2.31034E+00,5.49596E+01,4.64659E+02,-8.05722E-02,4.13030E+00,1.00000E+00,9.64407E-02,0.00000E+00,1.53571E-01,
/Users/AnnaN/Desktop/bbr/inst/model/nonmem/basic/test_batch_params/2,,0.00000E+00,2.31034E+00,,4.64659E+02,-8.05722E-02,4.13030E+00,1.00000E+00,9.64407E-02,0.00000E+00,1.53571E-01,5.49596E+01
/Users/AnnaN/Desktop/bbr/inst/model/nonmem/basic/test_batch_params/3,no ext file contents,,,,,,,,,,,
/Users/AnnaN/Desktop/bbr/inst/model/nonmem/basic/test_batch_params/4,no ext file contents,,,,,,,,,,,
````

Message on line 109 has been changed to: 

```
log.Fatal("no subdirectories with corresponding ctl or mod files found")
```

Note: this could cause duplication if a file has both a .mod and .ctl file in the root directory.